slack_sdk/models/blocks/blocks.py

```diff
@@ -604,3 +604,34 @@ def _validate_title_length(self):
     @JsonValidator(f"author_name attribute cannot exceed {author_name_max_length} characters")
     def _validate_author_name_length(self):
         return self.author_name is None or len(self.author_name) < self.author_name_max_length
+
+
+class RichTextBlock(Block):
```

Comment:
I'm still having an issue with the parsing of this block. Looks like it needs to be added [here](https://github.com/slackapi/python-slack-sdk/pull/1431/files#diff-d814a0f6e797d3125d300815a8a6bf58181fc5b4d0a2589444a3e81cfde55909R72-R91).

Reply:
@seratch

Reply:
@tj-smith47 Thanks for pointing this out. Here is a PR to fix it: https://github.com/slackapi/python-slack-sdk/pull/1433 We will release a new version shortly